gulpfile.js

```diff
@@ -188,7 +188,12 @@ function unittestTask(done) {
     args: {
       coverage: !!argv.coverage
     }
-  }, done).start();
+  },
+  // https://github.com/karma-runner/gulp-karma/issues/18
+  function(error) {
+    error = error ? new Error('Karma returned with the error code: ' + error) : undefined;
```

Comment:
I think that works, but to be sure, can you push a change that makes a test to fail. I'm curious to see how the Travis build behaves and how errors are reported.